modules/API/pages/built-in-endpoints.adoc

```diff
@@ -32,6 +32,8 @@ These endpoints are simple diagnostic utilities, which respond with the followin
 
 .GET /echo Request and Response
 
+Note: 9000 port is for RESTPP server then in the below request it should not return message "Hello GSQL" , do we need to correct this? 
```

Comment:
You wrote an internal comment and question, not a proposed change.
Your comment is not clear: I don't know what you mean by "should" in "it should not return message 'Hello GSQL'."